hal/halx86/apic/rtctimer.c

```diff
@@ -33,7 +33,7 @@ ULONG
 RtcClockRateToIncrement(UCHAR Rate)
 {
     ULONG Freqency = ((32768 << 1) >> Rate);
-    return (1000000 + (Freqency/2)) / Freqency;
+    return (10000000 + (Freqency/2)) / Freqency;
```

Comment:
I certainly wouldn't mind a comment for this function, since I personally can't reason about this change at all.

Reply:
I added some comments. Hope that makes it clearer.

Reply:
Much, thanks!